tools/upgrade-deps/tests/cli.test.js

```diff
@@ -72,19 +74,17 @@ describe('talend-upgrade-deps', () => {
 	});
 
 	it('should support a --latest option', () => {
-		const tmp = getTmpDirectory('basic-latest', fixturePath);
+		const tmp = getTmpDirectory('basic-latest', fixturePath, lock);
 		const output = spawnSync('node', [bin, '--latest'], { cwd: tmp });
 		const err = output.stderr.toString();
 		if (err) {
 			console.error(err);
 		}
-		const tmpLock = yarnpkg.parse(readFileSync(path.join(tmp, 'yarn.lock')).toString());
+		const tmpLock = getLockContent(tmp, lock);
 		const logs = output.stdout.toString();
 		expect(logs).toContain('package.json using latest');
-		expect(logs).toContain('"chokidar": "^2.1.0" => "');
-		expect(logs).toContain('"react-dom": "^16.0.0" => "');
-		expect(logs).toContain('"react": "16.0.0" => "');
-		expect(logs).toContain('"react": "16.0.0" => "');
+		expect(logs).toContain('"react-dom": "^16.6.0" => "');
+		expect(logs).toContain('"react": "^16.6.0" => "');
```

Comment:
npm is way more strict on peer deps, so I have to update the fixture, so test content has changed:
align react and react-dom (because react-dom has peer on react which make react-dom 16.6 impossible to install with react.
So I have pinned chokidar and align react and react-dom